src/integrations/ActiveCampaign/constants.js

```diff
@@ -0,0 +1,13 @@
+const NAME = 'ActiveCampaign';
```

Comment:
What is the config name for Active campaign?

Reply:
it is 'ACTIVE_CAMPAIGN'